stl/inc/xutility

```diff
@@ -4378,6 +4378,28 @@ _OutIt _Copy_memmove(move_iterator<_InIt> _First, move_iterator<_InIt> _Last, _O
     return _Copy_memmove(_First.base(), _Last.base(), _Dest);
 }
 
+template <class _InIt, class _OutIt>
+_OutIt _Copy_memmove_common(_InIt _IFirst, _InIt _ILast, _OutIt _OFirst, _OutIt _OLast) noexcept {
+    const char* const _IFirst_ch = const_cast<const char*>(reinterpret_cast<const volatile char*>(_IFirst));
+    const char* const _ILast_ch  = const_cast<const char*>(reinterpret_cast<const volatile char*>(_ILast));
+    char* const _OFirst_ch       = const_cast<char*>(reinterpret_cast<volatile char*>(_OFirst));
+    const char* const _OLast_ch  = const_cast<const char*>(reinterpret_cast<const volatile char*>(_OLast));
+    const auto _Count            = static_cast<size_t>(_STD min(_ILast_ch - _IFirst_ch, _OLast_ch - _OFirst_ch));

```

Comment:
The build is failing due to lack of macro-defense parentheses here (and on line 4398 below):

```
FAILED: stl/CMakeFiles/msvcp_objects.dir/src/cond.cpp.obj 
"C:\PROGRA~2\Microsoft Visual Studio\2019\Preview\VC\Tools\MSVC\14.28.29304\bin\Hostx64\arm\cl.exe"  /nologo /TP -DCRTDLL2 -DNTDDI_VERSION=NTDDI_WIN8 -DSTRICT -DWIN32_LEAN_AND_MEAN -D_ALLOW_ITERATOR_DEBUG_LEVEL_MISMATCH -D_ARM_ -D_CRTBLD -D_CRT_DECLARE_NONSTDC_NAMES=1 -D_CRT_STDIO_ARBITRARY_WIDE_SPECIFIERS -D_DLL -D_HAS_OLD_IOSTREAMS_MEMBERS=1 -D_STL_CONCRT_SUPPORT -D_STL_WIN32_WINNT=0x0602 -D_VCRT_ALLOW_INTERNALS -D_VCRT_WIN32_WINNT=0x0602 -D_WIN32_WINNT=0x0602 -I"C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\VC\Tools\MSVC\14.28.29304\crt\src\vcruntime" -I"C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\VC\Tools\MSVC\14.28.29304\crt\src\concrt" -IC:\a\1\s\stl\inc /diagnostics:caret /W4 /WX /w14265 /w15038 /d1FastFail /guard:cf /Z7 /Gm- /Gy /Zp8 /std:c++latest /permissive- /Zc:threadSafeInit- /Zl /O2 /Os /GL /EHsc /showIncludes /Fostl\CMakeFiles\msvcp_objects.dir\src\cond.cpp.obj /Fdstl\CMakeFiles\msvcp_objects.dir\ /FS -c C:\a\1\s\stl\src\cond.cpp
C:\a\1\s\stl\inc\xutility(4387,61): error C2589: '(': illegal token on right side of '::'
    const auto _Count            = static_cast<size_t>(_STD min(_ILast_ch - _IFirst_ch, _OLast_ch - _OFirst_ch));
                                                            ^
```

I will push a change to unblock the build and tests.